src/quick-lint-js/error.h

```diff
@@ -260,12 +260,38 @@
       error_unexpected_control_character, { source_code_span character; },     \
       .error(QLJS_TRANSLATE("unexpected control character"), character))       \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_unexpected_characters_in_binary_number,                            \
+      { source_code_span characters; },                                        \
+      .error(QLJS_TRANSLATE("unexpected characters in binary literal"),        \
+             characters))                                                      \
+                                                                               \
   QLJS_ERROR_TYPE(                                                             \
       error_unexpected_characters_in_octal_number,                             \
       { source_code_span characters; },                                        \
       .error(QLJS_TRANSLATE("unexpected characters in octal literal"),         \
              characters))                                                      \
                                                                                \
+  QLJS_ERROR_TYPE(                                                             \
+      error_unexpected_characters_in_hex_number,                               \
```

Comment:
Nit: Alphabetically sort the errors by the error name. `error_unexpected_characters_in_hex_number` should come before `error_unexpected_characters_in_octal_number`, for example.